### PR TITLE
rooms.yml: add Luuklag to CHQ

### PR DESCRIPTION
**Pull request by tripleee**

### CODE REVIEW

**Comment by makyen** [comment]
To add a user to SD privileges, new users need to be added to both [users.yml](https://github.com/Charcoal-SE/SmokeDetector/blob/master/users.yml) and [rooms.yml](https://github.com/Charcoal-SE/SmokeDetector/blob/master/rooms.yml).

If the user has a MS account, as an MS admin, you can look on the user's ["user info" page](https://metasmoke.erwaysoftware.com/users/1533), which is linked from their entry in the [user data and roles list](https://metasmoke.erwaysoftware.com/users?utf8=%E2%9C%93&filter=luuklag). In the "user info" page in the SE Auth block on top left there are lines labeled "users.yml:" and "rooms.yml". The text on those lines after the `:` is what should be added to each file.

**Comment by tripleee** [comment]
I hope the change to `users.yml` is correct; I found the instructions in the comments to be somewhat bewildering.

**Comment by makyen** [comment]
CI is failing on Circle, which is expected, but why is it passing on Travis? It shouldn't be passing on Travis.

File: rooms.yml [hunk 1]
```diff
@@ -158,6 +158,7 @@ stackexchange.com:
       - 17078244 # Daniil Manokhin
       - 8629256  # M--
       - 15062967 # Adrian Mole
+      - 195522  #luuklag
```
Comment (makyen) [inline rooms.yml:161]: The number here needs to be the user's global account ID, which is the first column in users.yml. In this case, that number is 1599858.

**Comment by makyen** [comment]
BTW: When adding SD privileges for a user for the first time, I usually send them one of the following text in chat (as multiple chat messages, given that there's some Markdown links). If I've already given them some of the information, I'll adapt the text based on what I've already said.

Add user to SD (user has existing MS account) [If you prefer to just point the user at the last time I used this text, it's [here](https://chat.stackexchange.com/transcript/11540?m=54072182#54072182)]:
```
I've added you to the privilege list for SD. You should run the `!!/amiprivileged` command to verify. It would be a good idea to read through the [Feedback Guidance](//charcoal-se.org/smokey/Feedback-Guidance),  [SmokeDetector wiki](//charcoal-se.org/smokey/), and [SmokeDetector commands](//github.com/Charcoal-SE/SmokeDetector/wiki/Commands) (in particular [feedback](//github.com/Charcoal-SE/SmokeDetector/wiki/Commands#privileged-commands-as-reply)).
If you haven't already read it, there's an [Introduction to Participating](https://charcoal-se.org/training/index) in Charcoal.
You may also want to look at the [userscripts we have available](//git.io/vDlAO) ([GitHub](//git.io/vXsZu)), as they can be quite helpful, depending on how you want to participate. For viewing, feedback & flagging in chat, I find [FIRE](//github.com/Charcoal-SE/userscripts/blob/master/fire/fire.user.js) and [AIM](//github.com/Charcoal-SE/userscripts/blob/master/autoflagging/autoflagging.user.js) to be the ones I use the most.
For flagging through the userscripts, your metasmoke account needs to be write authenticated with SE, which, if you haven't already done so, can be done from the [authentication page](https://metasmoke.erwaysoftware.com/authentication/status) and doesn't require you to also sign up for autoflagging, although authenticating is also a required step for signing up for autoflagging.
In addition, to enable manual flagging, the checkbox that says "Use my account to cast flags automatically" on [this page](https://metasmoke.erwaysoftware.com/flagging/preferences) also needs to be checked. If you have no entries in the "`ID        Site Count        Flags Used Today`" list at the bottom of that page, then autoflagging will still not be enabled, but manual flagging through the userscripts will be enabled.
Just so you know, we normally tell people when they are getting privileges that we practice the "ton of bricks" discipline method here: If you mess-up sufficiently, or consistently, we come down like a ton of bricks :-).
To [quote ArtOfCode](//chat.stackexchange.com/transcript/11540?m=51901405#51901405): "TL;DR: ton-of-bricks is mostly to say that we almost never need to enforce things because things enforce themselves through having a good community here. If it gets to the point that someone's enforcing something, something's gone badly wrong."
```

Add new user to SmokeDetector (user doesn't yet have an MS account):
```
I've added you to the privilege list for SD. You should run the `!!/amiprivileged` command to verify. It would be a good idea to read through the [Feedback Guidance](//charcoal-se.org/smokey/Feedback-Guidance),  [SmokeDetector wiki](//charcoal-se.org/smokey/), and [SmokeDetector commands](//github.com/Charcoal-SE/SmokeDetector/wiki/Commands) (in particular [feedback](//github.com/Charcoal-SE/SmokeDetector/wiki/Commands#privileged-commands-as-reply)).
If you haven't already read it, there's an [Introduction to Participating](https://charcoal-se.org/training/index) in Charcoal.
You may also want to look at the [userscripts we have available](//git.io/vDlAO) ([GitHub](//git.io/vXsZu)), as they can be quite helpful, depending on how you want to participate. For viewing, feedback & flagging in chat, I find [FIRE](//github.com/Charcoal-SE/userscripts/blob/master/fire/fire.user.js) and [AIM](//github.com/Charcoal-SE/userscripts/blob/master/autoflagging/autoflagging.user.js) to be the ones I use the most.
If you don't already have one, you will need a metasmoke (MS) account with both the "Flagger" and "Reviewer" roles to use the userscripts for feedback and/or flagging. For flagging through the userscripts, your metasmoke account needs to be write authenticated with SE, which, if you haven't already done so can be done from the [authentication page](https://metasmoke.erwaysoftware.com/authentication/status) and doesn't require you to also sign up for autoflagging, although authenticating is also a required step for signing up for autoflagging.
In addition, to enable manual flagging, the checkbox that says "Use my account to cast flags automatically" on [this page](https://metasmoke.erwaysoftware.com/flagging/preferences) also needs to be checked. If you have no entries in the "`ID        Site Count        Flags Used Today`" list at the bottom of that page, then autoflagging will still not be enabled, but manual flagging through the userscripts will be enabled.
Just so you know, we normally tell people when they are getting privileges that we practice the "ton of bricks" discipline method here: If you mess-up sufficiently, or consistently, we come down like a ton of bricks :-).
To [quote ArtOfCode](//chat.stackexchange.com/transcript/11540?m=51901405#51901405): "TL;DR: ton-of-bricks is mostly to say that we almost never need to enforce things because things enforce themselves through having a good community here. If it gets to the point that someone's enforcing something, something's gone badly wrong."
```

File: users.yml [hunk 2]
```diff
@@ -62,6 +62,7 @@
 1435254: [1435254, 262693, 347191, 4826457]  # Suraj Rao
 1445276: [1445276, 145086, 284827, 1364007]  # Wai Ha Lee
 1464473: [1464473, 40662, 352986, 1377770]  # Frenzy Li
+1599858: [1599858, 195522, 361484, 1481116]  # Luuklag
```
Comment (makyen) [inline users.yml:65]: This entry looks correct.

**Comment by tripleee** [comment]
Rebased & force pushed. Sorry for being dense.

**Comment by tripleee** [comment]
Tangentially I pushed https://github.com/Charcoal-SE/SmokeDetector/commit/64d5c3442b30de9ca1d649d0d800039d8be11e2e which adds a couple of comments to hopefully guide future editors. It felt too minor for a PR but noting it here for a bit of visibility.

**Comment by makyen** [comment]
This looks good now.

The issues were no problem. I didn't think you were dense. It definitely took me a few commits after users.yml was created to figure out exactly what was needed.

Thanks for adding the additional comments. The entries in the user info on MS are relatively recent. I forgot to add any mention of them in users.yml after I added them to MS.

**Pull request merged**